src/WebJobs.Script/Config/ScriptSettingsManager.cs

```diff
@@ -93,5 +98,25 @@ internal static IConfigurationBuilder CreateDefaultConfigurationBuilder()
                 .AddJsonFile("appsettings.json", optional: true)
                 .Add(new ScriptEnvironmentVariablesConfigurationSource());
         }
+
+        public static Tuple<IServiceProvider, IServiceCollection> CreateDefaultConfigurationWithHostJsonFileAndEnvBuilder(ScriptApplicationHostOptions applicationHostOptions, IEnvironment environment, ILoggerFactory loggerFactory, IMetricsLogger metricsLogger)
```

Comment:
Looks like you're using this only in a single place in tests. Hoping we can not add it to this Type, and instead build the test host as needed in the test. If we need to do it in multiple places, then add a test helper for it, but didn't seem like it.

Reply:
Sure. I'm removing from it. I just followed the old code's structure. :)

Reply:
Removed from the class.